### ISSUE TITLE
Cannot read properties of undefined (reading 'quips')

### ISSUE THREAD

**Issue by romimalta** [tloncorp/landscape-apps#2590]
I was updating a entry in internal comms group and now every time I try to scroll down in order to find it, this happens 

https://github.com/tloncorp/landscape-apps/assets/748181/093e3ffa-0cba-45a1-92de-c3d8d5d508f1

```
TypeError: Cannot read properties of undefined (reading 'quips')
    at i7 (https://natnex-ronret.tlon.network/apps/groups/assets/index.45654692.js:48:228663)
    at Kl (https://natnex-ronret.tlon.network/apps/groups/assets/react-beautiful-dnd.fde4d763.js:19:52884)
    at mf (https://natnex-ronret.tlon.network/apps/groups/assets/react-beautiful-dnd.fde4d763.js:23:8723)
    at pf (https://natnex-ronret.tlon.network/apps/groups/assets/react-beautiful-dnd.fde4d763.js:23:947)
    at yg (https://natnex-ronret.tlon.network/apps/groups/assets/react-beautiful-dnd.fde4d763.js:23:875)
    at rn (https://natnex-ronret.tlon.network/apps/groups/assets/react-beautiful-dnd.fde4d763.js:23:730)
    at sl (https://natnex-ronret.tlon.network/apps/groups/assets/react-beautiful-dnd.fde4d763.js:21:10884)
    at https://natnex-ronret.tlon.network/apps/groups/assets/react-beautiful-dnd.fde4d763.js:19:38716
    at e.unstable_runWithPriority (https://natnex-ronret.tlon.network/apps/groups/assets/react-beautiful-dnd.fde4d763.js:8:3811)
    at _r (https://natnex-ronret.tlon.network/apps/groups/assets/react-beautiful-dnd.fde4d763.js:19:38494)
```

in my phone works normally except that I find this in the middle of the entry:

![IMG_7427](https://github.com/tloncorp/landscape-apps/assets/748181/a5600adf-597b-4c59-a3e7-6d13696f4c54)

**Comment by romimalta** [comment]
Morning Patrick & team

For some reason (unknown to me)
it has started working normally again.

tysm,
r


On Jun 12, 2023, at 9:15 AM, patosullivan ***@***.******@***.***>> wrote:



Can't repro this one. Can I get access to the internal comms group?

**Comment by jamesacklin** [comment]
Fixed by #2630
